hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/test/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/fair/converter/TestFSQueueConverter.java

```diff
@@ -440,28 +454,27 @@ public void testQueueOrderingPolicy() throws Exception {
     rootQueue = fs.getQueueManager().getRootQueue();
 
     converter.convertQueueHierarchy(rootQueue);
-
     // root
     assertEquals("root ordering policy", null,
-        csConfig.get(PREFIX + "root.ordering-policy"));
+        capacitySchedulerConfig.get(PREFIX + "root.ordering-policy"));
```

Comment:
Could you please use the getAppOrderingPolicy() method to get the configs in this test case?